### DISCUSSION TITLE
NuGet package for version 1.0?

### DISCUSSION THREAD

**Discussion by duanqn** [microsoft/WindowsAppSDK#1931] [Q&A] [answered]
Seems a stupid question, but how can I upgrade to Windows App SDK 1.0 via NuGet packages?
I saw 1.0 was released a while ago but the latest NuGet package is still 0.8.6.
https://www.nuget.org/packages/Microsoft.ProjectReunion/

Is it released somewhere else?

**Comment by ghost1372** [comment, marked as answer]
you should look for windows app sdk
https://www.nuget.org/packages/Microsoft.WindowsAppSDK/

**Reply by duanqn** [reply]
OMG... thx 👍
